test/e2e/operator/prepare.go

```diff
@@ -0,0 +1,64 @@
+package operator
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+)
+
+func prepare(t *testing.T) (*framework.TestCtx, string) {
+	By("Initialize cluster resources")
+	cleanupRetryInterval := time.Second * 1
+	cleanupTimeout := time.Second * 5
+	ctx := framework.NewTestCtx(t)
+	err := ctx.InitializeClusterResources(&framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
+	Expect(err).ToNot(HaveOccurred(), "cluster resources not initialized")
+
+	// get namespace
+	By("Check operator is up and running")
+	namespace, err := ctx.GetNamespace()
+	Expect(err).ToNot(HaveOccurred())
+	err = waitForDeployment(t, framework.Global.KubeClient, namespace, time.Second*5, time.Second*90)
```

Comment:
We can use operator-sdk `WaitForDeployment` one it does not have `WaitForDaemonSet` that' why we copied here and modified

[1] https://github.com/operator-framework/operator-sdk/blob/master/pkg/test/e2eutil/wait_util.go#L43